sktime/transformations/series/summarize.py

```diff
@@ -32,14 +32,31 @@ class WindowSummarizer(BaseTransformer):
     lag_feature: dict of str and list, optional (default = dict containing first lag)
         Dictionary specifying as key the type of function to be used and as value
         the argument `window`.
-        For all keys other than `lag`, the argument `window` is a length 2 list
-        containing the integer `lag`, which specifies how far back
-        in the past the window will start, and the integer `window length`,
-        which will give the length of the window across which to apply the function.
-        For ease of notation, for the key "lag", only a single integer
+
+        For all keys other than `lag`, the argument `window` is a length 2 list.
+        The window elements can be either integers indicate the number of
```

Comment:
Suggestion:

```
Either elements of the window list can be one of:

* an integer, in which case it will indicate a difference in number of observations
* a string specifying a time period, see below - only for data `X` with `pd.DatatimeIndex` or `pd.TimedeltaIndex`
* an time difference object of type `pd.Timedelta`

Reply:
(this is better structured and makes the options clearer)

Reply:
agreed!